### PR TITLE
[Security] Clean up

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@Arminek This is great work, but also a BC break, could you document the changes needed in the UPGRADE? (config files and maybe note about how to upgrade routing)

File: src/Sylius/Bundle/ShopBundle/Resources/private/js/sylius-variants-prices.js [hunk 1]
```diff
@@ -33,11 +33,13 @@ function handleProductOptionsChange() {
...
 
         if ($price !== undefined) {
+            console.log('elo');
             $('#product-price').text($price);
-            $('#sylius-product-adding-to-cart').enable();
+            $('button[type=submit]').removeAttr('disabled');
         } else {
             $('#product-price').text($('#sylius-variants-pricing').attr('data-unavailable-text'));
-            $('#sylius-product-adding-to-cart').disable();
+            console.log('nope');
```
Comment (tuka217) [inline src/Sylius/Bundle/ShopBundle/Resources/private/js/sylius-variants-prices.js]: Is it needed?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/ShopBundle/Resources/private/js/sylius-variants-prices.js]: :D

Comment (Arminek) [inline src/Sylius/Bundle/ShopBundle/Resources/private/js/sylius-variants-prices.js]: 🌮

**Comment by pjedrzejewski** [comment]
Great, thank you Arek! :)

**Comment by smashou** [comment]
Could you, please, just explain or link the documentation which explain why  the locale switch and currency switch require now POST and not GET ? 
Just for my comprehension.

**Comment by michalmarcinkowski** [comment]
@smashou the idea was to protect it from XSS, but it's an overkill so we reverted that change in https://github.com/Sylius/Sylius/pull/7337.

**Pull request merged**